docs/source/reference/config.rst

```diff
@@ -287,6 +287,13 @@ Available fields and semantics:
               path: /tmp
               type: Directory
 
+    # Size of the /dev/shm shared memory for the pod (optional).
+    #
+    # Defaults to None, which means no size limits are set. If set, the value
+    # must be a string that is a valid Kubernetes quantity, e.g., "3Gi".
+    # https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/
```

Comment:
Could we mention the use case for specifying this in the comment?

Also, it seems a user could specify this directly in the `pod_config` above, is it necessary to have a separate field in the config for this?

Reply:
Added a comment. This is required by one of our users, since their production environment uses an AdmissionController that denies any pods which do not have a specific limit on /dev/shm size. 

Using pod_config is possible, but requires the user to understand our pod template and use the exact fields ( (`name: dshm, ...`)  when overriding the pod_config.

Reply:
Thanks for the explanation! How frequent this field will be used? I am wondering if it is not that frequently used, we can, instead, just have the full config in the `pod_config` in this doc, and have the comment there.
I kind of feeling including two ways to change the same config might be a bit confusing.

Reply:
Ahh good point. I thought more about it and indeed it likely won't be used that frequently. I added a comment to our config.yaml docs and updated the PR description.